Tests/FilesTests/FilesTests.swift

```diff
@@ -366,6 +366,30 @@ class FilesTests: XCTestCase {
             XCTAssertEqual(sequence.count, 6)
         }
     }
+
+    func testRenamingFoldersWhileEnumeratingSubfoldersRecursively() {
+        performTest {
+            let subfolder1 = try folder.createSubfolder(named: "1")
+            let subfolder2 = try folder.createSubfolder(named: "2")
+
+            try subfolder1.createSubfolder(named: "1A")
+            try subfolder1.createSubfolder(named: "1B")
+
+            try subfolder2.createSubfolder(named: "2A")
+            try subfolder2.createSubfolder(named: "2B")
+
+            let sequence = folder.makeSubfolderSequence(recursive: true)
+
+            for folder in sequence {
+                try! folder.rename(to: "Folder " + folder.name)
```

Comment:
You don't have to force-try here, since the enclosing closure is `throwing`. Force-trying in tests can be problematic when investigating CI failures, as you don't get a very informative error message 🙂